data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -302,6 +305,41 @@ def _load_score_csv(
             encoding="utf-8-sig",  # windows compat https://stackoverflow.com/a/43684587
         )
 
+    def _load_excel_from_df(
+        self, excel_df: pd.DataFrame, excel_path: Path
+    ) -> None:
+        # Define Excel Columns Column Width
+        NUM_EXCEL_COLS_WIDTH = 30
```

Comment:
What drives this number?

Reply:
This comes from the "Export File Changes" specs on our drive